src/pages/sangil/Login/Login.scss

```
@@ -0,0 +1,117 @@
+.form {
+  width: 310px;
+  height: 380px;
+  top: -50px;
+  left: -700px;
+  display: flex;
+  position: absolute;
+  flex-direction: column;
+  align-content: center;
+  flex-wrap: wrap;
+  background-color: #fff;
+  border: 1px solid #dbdbdb;
+}
+
+.form {
+  input {
```

현재 Login.js 를 보니 컴포넌트의 구성이 main_container > form_container > form 의 순서로 속해있네요!

```suggestion
.main_container {
  .form_container {
   .form {
     width: 310px;
     height: 380px;
    /* your code */
    }
  }
}
```
큰 클래스로 감싸주시는 건 어떨까 하고 생각해보았습니다!
다른 style 파일들도 마찬가지로 제안드려봅니다.
뒤에서는 관련 내용 생략하겠습니다!
상일님 , 클래스명 깔끔하게 작성해주신 것 같아요, 저도 참고해서 리팩토링 해보겠습니다 ^_^